server.go

```diff
@@ -214,6 +215,22 @@ func newFuncServerOption(f func(*serverOptions)) *funcServerOption {
 	}
 }
 
+// multiServerOption provides a way to combine arbitrary number of server
+// options into one.
+type multiServerOption struct {
```

Comment:
`joinServerOption`?  Or rename the other one.  It's confusing if these don't line up ~1:1.

Reply:
Done.